bigframes/operations/blob.py

```diff
@@ -37,7 +40,29 @@ def metadata(self):
         details_json = self._apply_unary_op(ops.obj_fetch_metadata_op).struct.field(
             "details"
         )
-
         import bigframes.bigquery as bbq
 
         return bbq.json_extract(details_json, "$.gcs_metadata")
+
+    def display(self, n: int = 3):
+        """Display the blob content in the IPython Notebook environment. Only works for image type now.
+
+        .. note::
+            BigFrames Blob is still under experiments. It may not work and subject to change in the future.
+
+        Args:
+            n (int, default 3): number of sample blob objects to display.
+        """
+        import bigframes.bigquery as bbq
+
+        s = bigframes.series.Series(self._block).head(n)
```

Comment:
relying on `peek` instead of `head` would make it relevant in partial ordering mode as well